data/resources/Urban_Jungle/terrain.cfg

```diff
@@ -37,7 +35,7 @@
         aliasof=Vjt
         def_alias=Vt
         mvt_alias=Mt
-        editor_group=eastern_invasion
+        editor_group=the_deceivers_gambit
```

Comment:
What? How on earth did this change happen? That should be `editor_group=urban_jungle`.

Reply:
Yeah, already fixed. I apologize for the mess